pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -11,10 +11,13 @@
     <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
       <TargetFramework>.NETCoreApp1.0</TargetFramework>
     </ProjectReference>
+    <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
+      <TargetFramework>.NETCoreApp1.1</TargetFramework>
+    </ProjectReference>
   </ItemGroup>
-
+  
   <Target Name="ReadDependenciesFromProjectJsonFile" BeforeTargets="GetPackageDependencies">
-    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)project.json">
+    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)\netcoreapp1.0\project.json">
```

Comment:
>I assume the package list is shared? If so then we shouldn't need to do this Regex hack twice. 

What do you mean by shared? The dependencies for netcoreapp1.0 are not equivalent to the dependencies for netcoreapp1.1 if that's what you mean. If they were then you're right, it would be one read and two TargetFramework tags.

>If we do need unique sets then we should just batch on the target frameworks so we don't need to duplicate the logic.

A batch would work. I honestly didn't put too much effort into it because it seemed like a band-aid solution at best. 

>PS: I wrote this regex hack and I really dislike it, it only exists because we didn't have a good way to have the package versions update with our update scripts. At some point we will fix that in core-setup and we can kill this hack.

That would be nice :) I originally tried adding the dependencies for netcoreapp1.1 under the dependencies of the framework, but I noticed they were getting applied to every framework. Then I found the Regex. Ideally we would have some real json parsing to read the dependencies under a framework and associate them with that framework instead of just reading every line and Regex'ing them. That way we would only need one project.json file and we could circumvent the need for duplication or a batch script.